sample-session-files/fake.appiumsession

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
I would rather make it versioned. It might be necessary to alter/change this format in the future, so we must know which version is supported and which is not (or probably needs to be upgraded)

Reply:
I updated `getSaveableState` to include version as a property.